index.js

```diff
@@ -205,6 +205,13 @@ const legacyScriptingRunner = (Zap, zobj, app) => {
   ) => {
     options = _.extend(
       {
+        // Options to deal with the response when post method is not defined
+        // * checkResponseStatus: throws an error if response status is not 2xx
+        // * parseResponse: assumes response content is JSON and parse it
+        // * ensureArray: could be one of the following values:
+        //   - false: returns whatever data parsed from response content
+        //   - 'wrap': returns [obj] if response is an object
+        //   - 'first': returns the first top-level array in the response if response is an object
```

Comment:
We should probably say `'first'` is the "default" value if it's not `false` nor `'wrap'`?